tacacs_plus/authorization.py

```diff
@@ -149,5 +153,6 @@ def __str__(self):
         return ', '.join([
             'status: %s' % self.human_status,
             'server_msg: %s' % self.server_msg,
+            'data: %s' % self.data,
```

Comment:
I am using it only for a test (test_authorization_reply_unpack_data) and it is defined in the protocol.. Do you think it should be removed?